test/run_dependency_with_broken_root/gleam.toml

```diff
@@ -0,0 +1,20 @@
+name = "run_dependency_with_broken_root"
+version = "1.0.0"
+
+# Fill out these fields if you intend to generate HTML documentation or publish
+# your project to the Hex package manager.
+#
+# description = ""
+# licences = ["Apache-2.0"]
+# repository = { type = "github", user = "username", repo = "project" }
+# links = [{ title = "Website", href = "https://gleam.run" }]
+#
+# For a full reference of all the available options, you can have a look at
+# https://gleam.run/writing-gleam/gleam-toml/. 
+
+[dependencies]
+gleam_stdlib = "~> 0.34 or ~> 1.0"
+birdie = "~> 1.0"
```

Comment:
This is a lot more code to compile than we need to, which makes it slower to run and introduces more chances of breakages. Let's run gleeunit in the tests instead as it is smaller.